Tests/Unit/RealMiniAppTests.swift

```diff
@@ -743,6 +743,20 @@ class RealMiniAppTests: QuickSpec {
                     expect(testResponseData).toEventually(beAnInstanceOf(RealMiniAppView.self), timeout: .seconds(10))
                 }
             }
+            context("when mini app send string to host app and delegate not implemented") {
+                it("will receive the error") {
+                    var testResponseError: UniversalBridgeError?
+                    realMiniApp.sendJsonToHostApp(info: "Test String to host from miniapp") { (result) in
+                        switch result {
+                        case .success:
+                            break
```

Comment:
optional: instead of `break` you could also use `fail("should not succeed")` so the failure message is clear

Reply:
yes I have updated this and will be pushed in next commit.